app/serializers/category_detailed_serializer.rb

```diff
@@ -10,7 +10,8 @@ class CategoryDetailedSerializer < BasicCategorySerializer
              :topics_year,
              :topics_all_time,
              :is_uncategorized,
-             :subcategory_ids
+             :subcategory_ids,
+             :subcategory_list
```

Comment:
Should we be using a serializer for the categories here?

Reply:
This will still need to use the same category serializer otherwise we're sending every single column on the `categories` table. Some may be sensitive information that should not be exposed to user. Also, we should exclude the `subcategory_list` attribute from the payload if the option is not present. Otherwise the following is very confusing where the `subcategory_ids` attribute contain values but the `subcategory_list` attribute is `null`. 

![Screenshot from 2021-10-05 11-36-52](https://user-images.githubusercontent.com/4335742/135956237-19676954-c481-48d6-a164-e1511d2d1008.png)